deploy/aws/boot.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Check if Lock File exists, if not create it and set trap on exit
+if { set -C; 2>/dev/null >/home/ubuntu/.appsmith.lock; }; then
+    trap "rm -f /home/ubuntu/.appsmith.lock" EXIT
+else
+    exit
+fi
+
+start_docker() {
+    if [ `systemctl is-active docker.service` == "inactive" ];then
+        echo "Starting docker"
+        `systemctl start docker.service`
+    fi
+}
+
+
+# generate random string
+generate_random_string() {
+    value=`cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 10 | head -n 1`
+    echo $value
+}
+
+install_dir="/home/ubuntu/appsmith/"
+
+if [ ! -d $install_dir ];then
+    mkdir -p $install_dir
+fi
+chown -R ubuntu:ubuntu $install_dir
+
+mongo_host="mongo"
+mongo_database="appsmith"
+mongo_root_user=$( generate_random_string )
+mongo_root_password=$( generate_random_string )
+user_encryption_password=$( generate_random_string )
+user_encryption_salt=$( generate_random_string )
+
+custom_domain=""
+NGINX_SSL_CMNT=""
+if [[ -z $custom_domain ]]; then
+    NGINX_SSL_CMNT="#"
+fi
+
+script_dir="/script"
+if [ ! -d $install_dir$script_dir ];then
+    mkdir -p $install_dir$script_dir
+fi
```

Comment:
Since we are doing `mkdir -p`, we shouldn't need to check if the folder exists. The `mkdir -p` command won't complain if the folder already exists.

```suggestion
mkdir -p "$install_dir/$script_dir"
```

Reply:
Removed check.